ludwig/schema/features/augmentation/base.py

```diff
@@ -0,0 +1,16 @@
+from ludwig.api_annotations import DeveloperAPI
+from ludwig.schema import utils as schema_utils
+
+
+@DeveloperAPI
+class BaseAugmentationConfig(schema_utils.BaseMarshmallowConfig):
+    """Base class for augmentation."""
+
+    pass
+
+
+@DeveloperAPI
+class BaseAugmentationContainerConfig(schema_utils.BaseMarshmallowConfig):
```

Comment:
I think this class will end up being unnecessary, can be removed

Reply:
commented out, added TODO to clean before merge